### ISSUE TITLE
[Bug]: Accessibility - Many visual cues on the email list don't have an accessible label

### ISSUE THREAD

**Issue by Almost-Senseless-Coder** [bulwarkmail/webmail#1006]
### Description

The unread dot, attachment, replied to, forwarded, etc. icons don't have an accessible label. Some do have a `title` attribute, but that's insufficient for most screen readers: The `title` attribute gets ignored in most settings, and the icons are `aria-hidden` anyways.

Additionally, in the most compact UI setting, the button to select an email isn't visible to screen readers, and in the unified mailbox, account labels don't get rendered to screen readers, either.

### Steps to Reproduce

1. Launch a screen reader, or accessibility tree inspector.
2. Open the unified mailbox or a regular mailbox, ideally in the most compact UI setting

### Expected Behavior

The icons mentioned above should render a textual representation to screen readers, e.g., "Select email checkbox; Unread; Joe Doe; Just an example mail With attachment"

### Actual Behavior

A lot of the information mentioned above is lost: Depending on your UI setting, the selection button might be missing, the unread dot and attachment icon don't render anything, and in the unified mailbox it's impossible to know which account a given email originated from.

### Bulwark Version

main

### Stalwart Mail Server Version

0.16.19

### Operating System

None

### Relevant Logs or Error Output

```shell

```
